local/Plugin/RcauthSource/Lib/lang.php

```diff
@@ -54,4 +55,6 @@
 	'pl.rcauthsource.issuer.desc'    => 'The DN of the certificate issuer',
 	'pl.rcauthsource.idphint'        => 'IdpHint parameter',
 	'pl.rcauthsource.idphint.desc'   => 'Optionally the VO portal can redirect the user to a specific IdP by also sending an idphint parameter',
+	'pl.rcauthsource.mp_oa2_server'        => 'Masterportal Oauth2 Server',
```

Comment:
Is this the issuer of the MasterPortal OP or the url of its well-known configuration?

Reply:
The latter. The url of its well-known configuration